Example/AblyChatExample/ContentView.swift

```diff
@@ -149,6 +150,16 @@ struct ContentView: View {
         }
     }
 
+    func printConnectionStatusChange() async throws {
```

Comment:
It awaits nothing here since the code wrapped in `Task`, btw why?

Reply:
it for awaits status in connectionSubsciption a few lines below? 

it was however not throwing anything so removed that at least... fair point on the Task wrapping... I'm sorting this out in an upcoming PR. None of those lines should need to be run in separate tasks